helix-lock/src/main/java/org/apache/helix/lock/LockInfo.java

```diff
@@ -65,34 +77,70 @@ public LockInfo(ZNRecord znRecord) {
     if (znRecord != null) {
       String ownerId = znRecord.getSimpleField(LockInfoAttribute.OWNER.name());
       String message = znRecord.getSimpleField(LockInfoAttribute.MESSAGE.name());
-      long timeout = znRecord.getLongField(LockInfoAttribute.TIMEOUT.name(), DEFAULT_TIMEOUT_LONG);
-      setLockInfoFields(ownerId, message, timeout);
+      long timeout = znRecord
```

Comment:
It would still be cleaner to just do a simple check instead of initializing the values 2 times.
For instance,
if (znRecord == null) {
  znRecord = new znRecord();
}
...
Then the default value will for sure be set in the same way since this record is empty.